### ISSUE TITLE
Remapping keys to a command that requires input

### ISSUE THREAD

**Issue by Eloitor** [helix-editor/helix#5499]
Hi,

I want to send lines from helix to tmux. For this I wrote this script:

```bash
$ cat tmux-send-stdin.sh
#!/bin/sh

# Get the session name from the command line argument
session="$1"

while read -r command; do
    tmux send-keys -t "$session" "$command" Enter
done
```

I wanted to create a map for `shell_pipe_to` such as

```
[keys.normal]
A-a = ["shell_pipe_to", "tmux-send-stdin.sh session-0"] 
```

output when running `hx`: 
```
Bad config: No command named 'tmux-send-stdin.sh session-0' for key `keys.normal.A-a` at line 23 column 1
Press <ENTER> to continue with default config
```

The main problem is that `shell_pipe_to` requires the user to enter the program to use, but I don't know if this is possible to emulate with a keymap. I found this similar use case: https://www.reddit.com/r/HelixEditor/comments/xde3f1/remapping_keys_to_a_command_that_requires_input/

Is this possible to do currently? Is this planned? Is there a better solution to send commands to tmux?

**Comment by archseer** [comment]
This is why `:pipe-to` was added. You should be able to map to `:pipe-to tmux-send-stdin.sh session-0`

**Comment by Eloitor** [comment]
Thanks for the answer. I get 

```
Bad config: No TypableCommand named ':pipe-to tmux-send-stdin.sh session-0' for key `keys.normal.A-a` at line 18 column 7
Press <ENTER> to continue with default config
```

I have version 22.12

**Comment by archseer** [comment]
Looks like it's not in a release yet, it was merged after 22.12: https://github.com/helix-editor/helix/pull/4931

**Comment by Eloitor** [comment]
While I'm waiting for the release I found this solution:

```
$ cat tmux-send-stdin.sh
#!/bin/sh

# Get the session name from the command line argument
session="$1"

while read -r command; do
    echo "$command"
    tmux send-keys -t "$session" "$command" Enter
done
```

And in the config:

```
[keys.normal]
A-a = ":pipe tmux-send-stdin.sh session-0"
```

**Comment by gabydd** [comment]
Can we close this as it's already solved with pipe-to plus a duplicate of https://github.com/helix-editor/helix/issues/1383 for keys that require input

**Comment by the-mikedavis** [comment]
Closed by #4931
